### ISSUE TITLE
support command: `SHOW INDEXES`

### ISSUE THREAD

**Issue by neverchanje** [risingwavelabs/risingwave#9046]
### Is your feature request related to a problem? Please describe.

# PostgreSQL `SHOW INDEXES` Command

The PostgreSQL `SHOW INDEXES` command is used to display a list of all
indexes that exist on a particular table in the current database. An
index is a database object that is used to improve the performance of
queries by allowing them to access data more quickly.

The basic syntax for the `SHOW INDEXES` command is as follows:

``` sql
SHOW INDEXES FROM table_name;
```

Here is a brief explanation of each element:

  - `table_name`: This is the name of the table for which you want to
    display the indexes.

If you execute the `SHOW INDEXES` command with a table name, RisingWave
will display a list of all indexes that exist on that table. The output
will include information about the name of the index, the name of the
table it belongs to, the type of index (such as btree or hash), and the
columns that the index is based on.

Here is an example of how to use the `SHOW INDEXES` command to display
the indexes for a table named `employees`:

``` sql
SHOW INDEXES FROM employees;
```

This command will display a list of all indexes that exist on the
`employees` table, including their names, types, and columns.

In summary, the `SHOW INDEXES` command is a useful tool for displaying
information about the indexes that exist on a particular table in a
PostgreSQL database. By using this command, you can get a better
understanding of the performance characteristics of your database and
make informed decisions about how to optimize your queries.

**Comment by chenzl25** [comment]
> Hi @chenzl25,
> 
> Could you please help implement this command since you developed the index-related features? If you don't have time, feel free to reassign this task to someone else or uncheck it.
> 
> Thank you!

Sure. I will implement it in this version.

**Comment by chenzl25** [comment]
But `SHOW INDEXES FROM table_name` is more like a syntax from `MySQL` instead of `PostgreSQL`.
